lambda-function/src/utils/measureConversionUtils.ts

```diff
@@ -118,7 +118,9 @@ const convertMeasureMetadata = (measureDetails: MeasureDetails): MeasureMetadata
     clinicalRecommendation: measureDetails.clinicalRecomms,
     references: references,
     endorsements: [endorsement],
-    definition: measureDetails.definitions,
+    measureDefinitions: !_.isEmpty(measureDetails.definitions)
+      ? [{ term: "", definition: measureDetails.definitions || "" }]
```

Comment:
should we create an `id` as well either here or in the transfer controller/service?

Reply:
I was thinking of adding it to transfer controller, Since we have models already. I will create a PR for that in few minutes